### PR TITLE
feat(1283): custom doc link

### PR DESCRIPTION
**Pull request by DekusDenial**
## Context
Customize the doc link

## Objective
Have UI overriding the default doc link with whatever from env var if there is such

## Related links
Related to:
- https://github.com/screwdriver-cd/screwdriver/issues/1283

### CODE REVIEW

File: config/environment.js [hunk 1]
```diff
@@ -49,6 +49,7 @@ module.exports = (environment) => {
       SDAPI_NAMESPACE: 'v4',
       SDSTORE_HOSTNAME: 'http://localhost:8081',
       SDSTORE_NAMESPACE: 'v1',
+      SDDOC_URL: 'http://docs.screwdriver.cd',
```
Comment (tkyi) [inline config/environment.js]: Maybe also have it configurable under prod environment (https://github.com/screwdriver-cd/ui/blob/master/config/environment.js#L94-L98)? Not sure what value that would bring though

File: nginx.conf [hunk 2]
```diff
@@ -63,7 +64,7 @@ http {
         # Dynamic configuration for the ember app
         location /assets/supplementary_config.js {
            content_by_lua_block {
-               ngx.say("window.SUPPLEMENTARY_CONFIG = { SDAPI_HOSTNAME: '", os.getenv("ECOSYSTEM_API"), "', SDSTORE_HOSTNAME: '", os.getenv("ECOSYSTEM_STORE"), "'  };")
+               ngx.say("window.SUPPLEMENTARY_CONFIG = { SDAPI_HOSTNAME: '", os.getenv("ECOSYSTEM_API"), "', SDSTORE_HOSTNAME: '", os.getenv("ECOSYSTEM_STORE"), "', SDDOC_URL: '", os.getenv("ECOSYSTEM_STORE"), "'  };")
```
Comment (tkyi) [inline nginx.conf]: should it be `os.getenv("SDDOC_URL")`?

Comment (DekusDenial) [reply nginx.conf]: oh well

File: config/environment.js (hunk 1 above)
Comment (DekusDenial) [reply config/environment.js]: actually, i was considering `http://docs.screwdriver.cd` as default since the cluster means to override it

**Pull request merged**